src/ApparentHorizons/ComputeExcisionBoundaryVolumeQuantities.tpp

```diff
@@ -0,0 +1,184 @@
+      "Found a dest tag that is not allowed");
+  static_assert(
+      std::is_same_v<tmpl::list_difference<required_dest_tags<Frame::Inertial>,
+                                           DestTagList>,
+                     tmpl::list<>>,
+      "A required dest tag is missing");
+
+  if (target_vars->number_of_grid_points() !=
+      src_vars.number_of_grid_points()) {
+    target_vars->initialize(src_vars.number_of_grid_points());
+  }
+
+  get<gr::Tags::SpacetimeMetric<3, Frame::Inertial>>(*target_vars) =
+    get<gr::Tags::SpacetimeMetric<3, Frame::Inertial>>(src_vars);
+}
+
+/// Dual frame case
+template <typename SrcTagList, typename DestTagList, typename TargetFrame>
+void ComputeExcisionBoundaryVolumeQuantities::apply(
+    const gsl::not_null<Variables<DestTagList>*> target_vars,
+    const Variables<SrcTagList>& src_vars, const Mesh<3>& /*mesh*/,
+    const Jacobian<DataVector, 3, TargetFrame, Frame::Inertial>& /*jacobian*/,
+    const InverseJacobian<DataVector, 3, Frame::ElementLogical, TargetFrame>&
+        /*inverse_jacobian*/) {
+
+  static_assert(
+      std::is_same_v<tmpl::list_difference<SrcTagList, allowed_src_tags>,
+                     tmpl::list<>>,
+      "Found a src tag that is not allowed");
+  static_assert(
+      std::is_same_v<tmpl::list_difference<required_src_tags, SrcTagList>,
+                     tmpl::list<>>,
+      "A required src tag is missing");
+
+  static_assert(
+      std::is_same_v<
+          tmpl::list_difference<DestTagList, allowed_dest_tags<TargetFrame>>,
+          tmpl::list<>>,
+      "Found a dest tag that is not allowed");
+  static_assert(
+      std::is_same_v<
+          tmpl::list_difference<required_dest_tags<TargetFrame>, DestTagList>,
+          tmpl::list<>>,
+      "A required dest tag is missing");
+
+  const auto& psi =
+      get<gr::Tags::SpacetimeMetric<3, Frame::Inertial>>(src_vars);
+  const auto& pi =
+      get<GeneralizedHarmonic::Tags::Pi<3, Frame::Inertial>>(src_vars);
+  const auto& phi =
+      get<GeneralizedHarmonic::Tags::Phi<3, Frame::Inertial>>(src_vars);
+
+  if (target_vars->number_of_grid_points() !=
+      src_vars.number_of_grid_points()) {
+    target_vars->initialize(src_vars.number_of_grid_points());
+  }
+
+  using metric_tag = gr::Tags::SpatialMetric<3, TargetFrame>;
+  using inv_metric_tag = gr::Tags::InverseSpatialMetric<3, TargetFrame>;
+  using lapse_tag = ::Tags::TempScalar<0, DataVector>;
+  using shift_tag = ::Tags::TempI<1, 3, Frame::Inertial, DataVector>;
+  using spacetime_normal_vector_tag =
+      ::Tags::TempA<2, 3, Frame::Inertial, DataVector>;
+
+  // Additional temporary tags used for multiple frames
+  using inertial_metric_tag = gr::Tags::SpatialMetric<3, Frame::Inertial>;
+  using inertial_inv_metric_tag =
+      gr::Tags::InverseSpatialMetric<3, Frame::Inertial>;
+  //using inertial_ex_curvature_tag =
+  //    gr::Tags::ExtrinsicCurvature<3, Frame::Inertial>;
+  using logical_deriv_metric_tag = ::Tags::TempTensor<
```

Comment:
We don't need derivatives on the excision boundary right now (we might at some point if we want to do high-order crossing predictions). Remove all derivative-related code.